CHANGELOG.md

```diff
@@ -59,6 +59,7 @@ All notable changes to this project will be documented in this file based on the
 ### Added
 
 * support for elasticsearch-php ^7.0
+* Added `\Elastica\ResultSet::getTotalHitsRelation()` to get relation for total hits [#1694](https://github.com/ruflin/Elastica/pull/1694)
```

Comment:
This line should be added to the "unreleased" changes, after line 10!

Reply:
Oups, I messed up with the previous changes, I fixed that

Reply:
@ruflin @thePanz do you think that can be merged now, as I will have to rebase my other PR as I changed many lines the changelog here